helloworld-mutual-ssl-secured/pom.xml

```diff
@@ -29,6 +29,15 @@
         <relativePath/>
     </parent>
 
+    <properties>
```

Comment:
this should be moved to where it was, after <licenses/>

Reply:
Moved!